addon/components/base/bs-button.js

```diff
@@ -449,4 +453,37 @@ export default class Button extends Component {
 
     return this.get('bubble');
   }
+
+  init() {
```

Comment:
I tried to write a decorator instead of putting it here directly but wasn't successful. Haven't expected that decorators are that hard. I had some solutions but they were breaking quickly if put in different context...